### PR TITLE
Update to v0.11.0

### PR DESCRIPTION
**Pull request by pp-mo**

### CODE REVIEW

**Comment by pelson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "0.10.1" %}
+{% set version = "0.11.0" %}
 
 package:
   name: iris-grib
```
Comment (pelson) [inline recipe/meta.yaml:4]: Doesn't this version of iris-grib require iris ``2.*``?

Comment (pp-mo) [reply recipe/meta.yaml:4]: Erm... yes.
Will address ...

**Comment by ocefpaf** [comment]
Missed this one and merged #7 

@pp-mo let me know if you need this version, if so I can create a branch add it there.

**Comment by pp-mo** [comment]
> @pp-mo let me know if you need this version, if so I can create a branch add it there.

@ocefpaf thanks for asking.
Yes, if you would, I think for completeness it would be safer, thanks.

Iris Travis testing is still using python-ecmwf_grib, but installs v0.11 with pip direct from github.
When we progress to eccodes and 0.12, I expect we'll get both from here.
However, if there are others needing to stick with the old gribapi, then availability of v0.11 on conda-forge would be relevant to them.

**Comment by ocefpaf** [comment]
@pp-mo please take a look at https://github.com/conda-forge/iris-grib-feedstock/pull/8 and https://github.com/conda-forge/iris-grib-feedstock/pull/9

**Pull request closed without merging**